src/index.ts

```diff
@@ -2,15 +2,14 @@ import fs from 'node:fs/promises';
 import path from 'node:path';
 import makeDir from 'make-dir';
 import { generatorHandler } from '@prisma/generator-helper';
-import { parseEnvValue, logger } from '@prisma/internals';
```

Comment:
I don't see why there is the need to get rid of `@prisma/internals`? It is installed anyway along with `@prisma/client`. Hence, `parseEnvValue` doesn't need to be implemented twice. And the prisma logger has the nice side effect that it prefixes the output with cyan-colored "prisma:info".

Can you elaborate more on the reasons why?

Reply:
Sure - I had both felt nervous about the local usage of it because it was leading to Prisma to issue a warning saying that it wasn't intended for external use and because I was having issues where it would deadlock my imports because of the CJS -> ESM -> CJS back and forth apparently causing problems for webpack's compiled imports.

That was, though, with the root package we're both forked off of (I wrote this PR for that package but then found your fork and decided to contribute here instead).

I do think `parseEnvValue` is easy enough to clone out and use directly. I can see if this fork has the same CJS/ESM conflict that the origin package had, if that's not causing runtime issues anymore for me I'm happy to switch back to it.